hera/character/Portrait.tsx

```diff
@@ -30,43 +29,64 @@ export default memo(function Portrait({
   unit: UnitInfo;
   variant?: number;
 }) {
+  const ref = useRef<HTMLCanvasElement>(null);
   const hasPortraits = useSprites('portraits');
-  const sprite = hasPortraits
-    ? spriteURL('Portraits', encodeDynamicPlayerID(player))
-    : '';
 
   const { position } = unit.sprite.portrait;
-  const y = -(position.y + (variant || 0)) * PortraitHeight + 'px';
-  const alternateY = -(position.y + (variant || 0) + 6) * PortraitHeight + 'px';
+  const keyFrameYs = [
```

Comment:
Maybe we can call this `positions`?

Reply:
[refactor: rename variables](https://github.com/nkzw-tech/athena-crisis/pull/32/commits/9331080cfe4ad8ea0b79525f39d727743eb70cfe)